### PR TITLE
Added Outfoxed board game

### PR DESCRIPTION
**Pull request by Asymtode712**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

**Comment by Asymtode712** [comment]
@edm00se Please review my PR and merge it if everything's right

File: readme.md [hunk 1]
```diff
@@ -2212,6 +2212,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 | ------: | -------: | -----: |
 |   2 - 4 |        6 | 30-35m |
 
+### [Outfoxed](https://boardgamegeek.com/boardgame/172931/outfoxed (board_game))
+
+> This game is like Clue but for younger kids – and yet still fun for adults too. A fox stole a pot pie, and you have to figure out which fox it was before they escape into the foxhole. My kids regularly play this cooperative whodunnit game independently, and I love watching them work together as a team to solve the mystery. This is a fun little game to build your child’s logic and deductive reasoning skills!
+
+![Outfoxed](https://happyyouhappyfamily.com/wp-content/uploads/2017/03/outfoxed-1-crop-250px.webp)
```
Comment (edm00se) [inline readme.md:2219]: This will not be an acceptable image source.

File: readme.md [hunk 2]
```diff
@@ -2212,6 +2212,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 | ------: | -------: | -----: |
 |   2 - 4 |        6 | 30-35m |
 
+### [Outfoxed](https://boardgamegeek.com/boardgame/172931/outfoxed (board_game))
```
Comment (edm00se) [inline readme.md:2215]: The link looks malformed.

```suggestion
### [Outfoxed](https://boardgamegeek.com/boardgame/172931/outfoxed)
```

**Review by edm00se — CHANGES_REQUESTED** [review]
Checking [the linter action's output](https://github.com/edm00se/awesome-board-games/actions/runs/6550561507/job/17789934829?pr=391#step:4:44) shows that it's failing, while the message seems slightly off, I believe it's not happy with the title. Additionally, the image source will not work, please use one from wikipedia or BGG.

**Comment by Asymtode712** [comment]
@edm00se I have done the changes as suggested by you. Please review my request

File: readme.md [hunk 3]
```diff
@@ -2212,6 +2212,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
 
+### [Outfoxed](https://boardgamegeek.com/boardgame/172931/outfoxed)
+
+> This game is like Clue but for younger kids – and yet still fun for adults too. A fox stole a pot pie, and you have to figure out which fox it was before they escape into the foxhole. My kids regularly play this cooperative whodunnit game independently, and I love watching them work together as a team to solve the mystery. This is a fun little game to build your child’s logic and deductive reasoning skills!
+
+![Outfoxed](https://upload.wikimedia.org/wikipedia/commons/f/f6/Outfoxed.webp)
+
+| Players | Min. Age | Time   |
+|-------: | -------: | ------:|
+|   2 - 4 |       4  |  15m   |
```
Comment (edm00se) [inline readme.md:2223]:
```suggestion
|   2 - 4 |       5  |  20m   |
```

Changing the stats to match those reported by BGG.

**Review by edm00se — APPROVED** [review]
Thanks for contributing!

refs #269

**Pull request merged**